CMakeLists.txt

```diff
@@ -26,6 +26,35 @@ endif()
 
 file(GLOB _files *.fdm_material *.sig)
 
+# Determine version number
+find_package(Git)
+
+set(GIT_VERSION "Unknown")
+if(GIT_FOUND)
+    execute_process(
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        COMMAND ${GIT_EXECUTABLE} describe --tags --exact-match --dirty
+        OUTPUT_VARIABLE GIT_VERSION
```

Comment:
I'd add `RESULTS_VARIABLE` to get the exit code, and only use the output variable if the exit code is 0. See https://cmake.org/cmake/help/v3.12/command/execute_process.html

Reply:
Used RESULT_VARIABLE instead :) Great suggestion.